socorro/external/boto/connection_context.py

```diff
@@ -214,7 +214,11 @@ def list_objects_paginator(self, bucket, prefix):
 
         """
         paginator = self.client.get_paginator("list_objects_v2")
-        page_iterator = paginator.paginate(Bucket=bucket, Prefix=prefix)
+        page_iterator = paginator.paginate(
+            Bucket=bucket,
+            Prefix=prefix,
+            PaginationConfig={} if page_size is None else {"PageSize": page_size},
+        )
```

Comment:
If we pass an empty dict here is that the same as the default `Paginationconfig` for `paginate`?